scripts/scil_tractogram_compute_TODI.py

```diff
@@ -91,12 +91,12 @@ def main():
                         [args.mask, args.reference])
     assert_headers_compatible(parser, args.in_tractogram, args.mask,
                               reference=args.reference)
+    assert_outputs_exist(parser, args, [],
+                         [args.out_mask, args.out_tdi, args.out_todi_sf,
+                          args.out_todi_sh])
 
-    out_files = [args.out_mask, args.out_tdi, args.out_todi_sf,
-                 args.out_todi_sh]
-    assert_outputs_exist(parser, args, [], out_files)
-
-    if not np.any(out_files):
+    if not (args.out_mask or args.out_tdi or args.out_todi_sf or
```

Comment:
Previous version meant less code duplication no? Adding a new output type would require modifying it at these two places whereas before we only needed to add it to the list.

Reply:
I'm not sure why, but when I added the additional output in the test, this line failed... I can test again.

Reply:
Modified.